src/Provider/CacheProvider.php

```diff
@@ -15,10 +15,19 @@
 
 final class CacheProvider extends ServiceProvider
 {
+    private static string $cachePath;
```

Comment:
Why making it static? There's only one instance of CacheProvider created...

Reply:
not to pass the parameter in the static function.

```php
        $container->set(CacheInterface::class, static function (ContainerInterface $container) {
            $aliases = $container->get(Aliases::class);

            return new FileCache($aliases->get(self::$cachePath));
        });
```